package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "run-all",
+  "version": "1.0.0",
+  "description": "## Project setup",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "docker-compose up & (cd api && npm start) & (cd my-vue3-app && npm run serve)"
```

Comment:
Do you really want add this file because of this string?
Better to move this only line to readme

Reply:
We still may create `cheat sheet . md` with instructions